docs/ops/detection/DetectionOutput_8.md

```diff
@@ -0,0 +1,198 @@
+## DetectionOutput <a name="DetectionOutput"></a> {#openvino_docs_ops_detection_DetectionOutput_8}
+
+**Versioned name**: *DetectionOutput-8*
+
+**Category**: *Object detection*
+
+**Short description**: *DetectionOutput* performs non-maximum suppression to generate the detection output using information on location and 
+confidence predictions.
+
+**Detailed description**: [Reference](https://arxiv.org/pdf/1512.02325.pdf). The layer has 3 mandatory inputs: tensor with box logits, 
+tensor with confidence predictions and tensor with box coordinates (proposals). It can have 2 additional inputs with additional confidence 
+predictions and box coordinates described in the [article](https://arxiv.org/pdf/1711.06897.pdf). The output tensor contains information 
+about filtered detections described with 7 element tuples: `[batch_id, class_id, confidence, x_1, y_1, x_2, y_2]`. The first tuple 
```

Comment:
I think you should use () instead of [] for tuple.

Reply:
personally, I like more to see `[]`. It tastes differ:)